concord-server/server/src/test/java/com/rabidgremlin/concord/resources/StatsResourceTest.java

```diff
@@ -106,17 +114,44 @@ public void shouldFilterOutBulkUploadUser()
   {
     // Given
     List<UserVoteCount> dummyVotes = Collections.singletonList(new UserVoteCount("BULK_UPLOAD", 9999));
-    when(statsDao.getCountOfTotalVotesPerUser()).thenReturn(dummyVotes);
+    when(userStatsDao.getCountOfTotalVotesPerUser()).thenReturn(dummyVotes);
 
     // When
     Response response = statsResource.getUserStats(caller);
 
     // Then
     assertThat(response, instanceOf(Response.class));
-    assertEquals(200, response.getStatus());
-    assertEquals("OK", response.getStatusInfo().toString());
+    assertThat(response.getStatus(), is(200));
+    assertThat(response.getStatusInfo().toString(), is("OK"));
     assertThat(response.getEntity(), instanceOf(List.class));
-    assertEquals(0, ((List) response.getEntity()).size());
+    assertThat(((List) response.getEntity()).size(), is(0));
+  }
+
+  @Test
+  public void shouldGetSystemStats()
+  {
+    // Given
+    when(systemStatsDao.getTotalCountOfPhrases()).thenReturn(100);
```

Comment:
For better readability, I'd assign all the numbers to local variables so you have a linkage between the input and out
i.e.
`int totoCountOfPhrases = 100`
`assertThat(response.getEntity(), is(new SystemStats(totoCountOfPhrases....`